Configure fails if --with-ft=none is used

## Background information
### What version of Open MPI are you using? (e.g., v4.1.6, v5.0.1, git branch name and hash, etc.)
5.0.7
### Describe how Open MPI was installed (e.g., from a source/distribution tarball, from a git clone, from an operating system distribution package, etc.)
Not installed yet (trying to build the tarball from the OMPI website)
### Please describe the system on which you are running
* Operating system/version: Ubuntu 20.04
* Computer hardware: Older workstation-like PC (i9-10940X, no fancy networking)
* Network type: 1 GbE (irrelevant, only want intra-node)
-----------------------------
## Details of the problem
Trying to run the configure script of OpenMPI with --with-ft=none (which is supposedly supported according to https://docs.open-mpi.org/en/main/installing-open-mpi/configure-cli-options/misc.html ) results in this error:
```
configure: ===== done with 3rd-party/prrte configure =====
checking for internal PRRTE RST files... not found
checking for prte.h... no

*** Fault tolerance
checking if want fault tolerance... Unrecognized FT TYPE: none
configure: error: Cannot continue
```
Please find attached the entire configure output.

[config.txt](https://github.com/user-attachments/files/20040376/config.txt)

--- a/docs/installing-open-mpi/configure-cli-options/misc.rst
+++ b/docs/installing-open-mpi/configure-cli-options/misc.rst
@@ -20,8 +20,8 @@ above categories that can be used with ``configure``:
 
 * ``--with-ft=TYPE``:
   Specify the type of fault tolerance to enable.  The only allowed
-  values are ``ulfm`` and ``none``.  See :ref:`the ULFM section
-  <ulfm-label>` for more details.
+  values are ``ulfm`` and ``no`` (the default value is ``no``).  See
+  :ref:`the ULFM section <ulfm-label>` for more details.
 
 * ``--enable-peruse``:
   Enable the PERUSE MPI data analysis interface.

